src/components/Matches/FileExporter/index.tsx

```diff
@@ -58,6 +59,10 @@ const FileExporter = ({
   if (focused && +focused !== 1 && customLocationDescription.taxonomy) {
     customLocationDescription.taxonomy.accession = focused;
   }
+  const counters = getNeededCountersForSubpages(secondary, primary, true);
```

Comment:
Adding a comment for future me :sweat_smile:

```suggestion
  /*
   * Not a mistake: the filtering/secondary endpoint in the client (subpage)
   * becomes the main endpoint when exporting data.
   * e.g. InterPro entry IPR000001 > Proteins
   *   client: `/entry/interpro/IPR000001/protein/uniprot`
   *   export: `/api/protein/uniprot/entry/interpro/IPR000001/`
   */
  const counters = getNeededCountersForSubpages(secondary, primary, true);
```